hack/cut-release.sh

```diff
@@ -97,7 +97,7 @@ When it's done you can merge. (Continue to next section while you're waiting...)
 Go to the integration tests at ${KONFLUX_APPS_URL}/${KONFLUX_APPLICATION_NAME}/integrationtests
 Edit ${KONFLUX_APPLICATION_NAME}-enterprise-contract and add a parameter as follows:
   Name: POLICY_CONFIGURATION
-  Value: github.com/enterprise-contract/config//redhat-no-hermetic
+  Value: github.com/enterprise-contract/ec-cli
```

Comment:
Hmm if this is meant for a release branch, we should probably use the same exact policy as it is used by the release pipeline.

Reply:
It effectively is the same now I think. But note that (IIUC) we can't access it from gitlab directly, so either way we would need to make a copy of it.

Reply:
It depends on who's we. Red Hatter's? Sure we can. It is also synced to the cluster so Konflux users also have access. I'm actually quite confused about what this script is doing. It looks like it's about configuring Konflux for a new release branch. If so, I strongly recommend using the same exact release policy for integration tests. Otherwise, we are bound to find surprises at release time.

Reply:
Script doesn't do anything by itself, it's really just a piece of interactive documentation.